Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.cpp

```diff
@@ -168,18 +168,17 @@ static MediaKeyStatus mediaKeyStatusFromOpenCDM(std::string& keyStatus)
     return MediaKeyStatus::InternalError;
 }
 
-static size_t messageLength(std::string& message, std::string& request)
+static RefPtr<SharedBuffer> cleanResponseMessage(std::string& message)
 {
-    size_t length = 0;
-    std::string delimiter = ":Type:";
-    std::string requestType = message.substr(0, message.find(delimiter));
-    // FIXME: What is with this weirdness?
-    if (requestType.size() && requestType.size() == (request.size() + 1))
-        length = requestType.size() + delimiter.size();
-    else
-        length = request.length();
-    GST_TRACE("delimiter.size = %u, delimiter = %s, requestType.size = %u, requestType = %s, request.size = %u, request = %s", delimiter.size(), delimiter.c_str(), requestType.size(), requestType.c_str(), request.size(), request.c_str());
-    return length;
+    // If message does not begin with "message:", we bail out empty.
+    if (message.compare(0, 8, "message:"))
```

Comment:
I'd be tempted to start factoring these constants out in preparation for moving to a sensible interchange format. It makes me uneasy seeing magic numbers that you have to make sure are really the length of raw strings. I'd at least introduce some variables like "messageKey" and "typeKey" so that we can at least rely on sizeof. This maybe slightly premature, since when we do introduce a format, all this code will thankfully be removed.